src/popup/router/index.ts

```diff
@@ -58,8 +58,14 @@ RouteQueryActionsController.init(router);
 RouteLastUsedRoutes.init(router);
 
 router.beforeEach(async (to, from, next) => {
+  // Move the route resolving to the next thread cycle in order to allow the app
+  // to access the stored data (localStorage/browserStorage) first.
+  await new Promise((resolve) => setTimeout(resolve, 0));
```

Comment:
does it make sense to use `nextTick()` from vue here?

Reply:
This looks like a workaround to me, then an actual good decision.